src/views/Nft/market/Collection/IndividualNFTPage/MoreFromThisCollection.tsx

```diff
@@ -90,6 +90,10 @@ const MoreFromThisCollection: React.FC<MoreFromThisCollectionProps> = ({ collect
     swiperRef.slideTo(index)
   }
 
+  const updateActiveIndex = () => {
+    setActiveIndex(Math.ceil(swiperRef.activeIndex / slidesPerView))
+  }
```

Comment:
Did you check the parameters passed in? Seems unlikely that a `onActiveIndexChange` handler wouldn't pass in the old and new.

Reply:
You were right, this fixed.